react/src/components/FullFeaturedCard.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from "react";
+import { loadAirwallex, createElement } from "airwallex-payment-elements";
+
+const intentid = "int_1OF0qMxLz3b8s8unOzIAMjT4qep";
```

Comment:
Before merge the PR, 
replace all `intentid` to `replace-with-your-intent-id`
replace all`client_secret` to `replace-with-your-client-secret`